app/views/mobile-client-builds-row.html

```diff
@@ -71,7 +71,7 @@
             <b>Build Platform</b>
           </div>
           <div class="col-sm-6 col-md-4">
-            
+            {{row.apiObject.metadata.labels['mobile-client-build-platform']}}
```

Comment:
I think there is a label filter

Reply:
https://github.com/aerogear/origin-web-common/blob/master/src/filters/resources.js#L94

Reply:
I don't think that filter works well within html. They sem to be doing the same [here]( https://github.com/openshift/origin-web-console/blob/master/app/views/overview/_virtual-machine-row.html#L75) so i think it should be fine.

Reply:
@JameelB  What result do you get with the label filter? Why doesn't it work well?

Reply:
@sedroche doesn't return anything for me. I haven't seen an example where they use it in html files, only on js.